backend/internal/db/schedule.go

```diff
@@ -0,0 +1,26 @@
+package db
+
+import (
+	_ "github.com/jak103/powerplay/internal/models"
+	"gorm.io/gorm"
+)
+
+// TODO - fix the models package import
+
+// GetIceTimes retrieves all ice times from the database
+func GetIceTimes(db *gorm.DB) ([]IceTime, error) {
+	var iceTimes []IceTime
+	result := db.Find(&iceTimes)
```

Comment:
Do we still have the iceTimes model? I thought you had removed that.

Reply:
No that is a function I made to get icetimes out of the db but now we know we arent doing that. the ref_schedule code is in ref.go